clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -149,20 +164,28 @@ public void testStartupAndTearDown() throws InterruptedException {
                 "The consumer network thread did not stop within " + DEFAULT_MAX_WAIT_MS + " ms");
     }
 
+    @Test
+    void testRequestManagersArePolledOnce() {
+        consumerNetworkThread.runOnce();
+        requestManagers.entries().forEach(rmo -> rmo.ifPresent(rm -> verify(rm, times(1)).poll(anyLong())));
```

Comment:
i believe times(1) is the default, could you verify this?

Reply:
It is, removing instances of times(1)